libraries/src/Console/VersionCommand.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Joomla! Content Management System
+ *
+ * @copyright  Copyright (C) 2005 - 2020 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\CMS\Console;
+
+\defined('JPATH_PLATFORM') or die;
+
+use Joomla\CMS\Version;
+use Joomla\CMS\Updater\Updater;
+use Joomla\Console\Command\AbstractCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Console command for showing current CMS version
+ *
+ * @since  4.0.0
+ */
+class VersionCommand extends AbstractCommand
+{
+	/**
+	 * The default command name
+	 *
+	 * @var    string
+	 * @since  4.0.0
+	 */
+	protected static $defaultName = 'core:version';
+
+	/**
+	 * Internal function to execute the command.
+	 *
+	 * @param   InputInterface   $input   The input to inject into the command.
+	 * @param   OutputInterface  $output  The output to inject into the command.
+	 *
+	 * @return  integer  The command exit code
+	 *
+	 * @since   4.0.0
+	 */
+	protected function doExecute(InputInterface $input, OutputInterface $output): int
+	{
+		$symfonyStyle = new SymfonyStyle($input, $output);
+
+		$symfonyStyle->title('Fetching current Joomla version');
+
+		// Instantiate the Version object
+		$version = new Version;
+
+		$symfonyStyle->text($version->getLongVersion());
+
+		$symfonyStyle->success('');
+
+		return 0;
+	}
+
+	/**
+	 * Configure the command.
+	 *
+	 * @return  void
+	 *
+	 * @since   4.0.0
+	 */
+	protected function configure(): void
+	{
+		$this->setDescription('Shows the installed version of Joomla');
```

Comment:
```suggestion
		$this->setDescription('Show the installed version of Joomla');
```
See PR #29799 for consistent grammar.

Reply:
not needed anymore